crypto/s2n_hkdf.h

```diff
@@ -20,6 +20,15 @@
 #include "crypto/s2n_hmac.h"
 #include "utils/s2n_blob.h"
 
+struct s2n_hkdf_impl {
```

Comment:
You can put this in s2n_hkdf.c, since it doesn't need to be visible to other modules.